lib/ansible/modules/cloud/azure/azure_rm_storageaccountkeys_facts.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2016 Amanvir Mundra, <@amanvirmundra>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_storageaccountkeys_facts
+
+version_added: "2.6"
+
+short_description: Retrieve Azure Storage Account Keys.
+
+description:
+    - Retrive keys for a Storage Account.
+
+options:
```

Comment:
One thing we are trying to do with the Azure facts modules is to return both a curated and raw/api structure as specified by a `format` option. We want to favour the `curated` values and set that as the default output which means they need to be in place before we merge the module.

The curated option is designed to flatten and make the return values from Azure a lot easier to view and access in Ansible.

Reply:
I am not sure I follow? It will be helpful if you can point me to where it is implemented in some other modules.